csp_billing_adapter/csp_hookspecs.py

```diff
@@ -74,5 +74,13 @@ def get_account_info(config: Config) -> dict:
     Retrieves the account information from the loaded plugin
 
     :param config: The application configuration dictionary
-    :return: Returns a dictionary of the account info from CSP
+    :return:
+        Returns a dictionary of the account info from CSP
+        in the following format:
+
+        {
+            'account_id': '123456789',
+            'arch': 'x86_64',
+            'cloud_provider': 'local'
+        }
```

Comment:
If we only have this data there is no way for us to verify that the data has not been manufactured.

Reply:
This requirement was lost or not spelled out in https://github.com/SUSE-Enceladus/csp-billing-adapter/issues/20. We need to clarify requirements there.

Reply:
Well in the conversation of #20 it states 

"We want to run the commands that are configured in the cloud regionsrv-client configuration and the information needs to be included in the suportconfig" 

those commands provide all the information. I am not certain, other than copy and pasting the commands on how that could have been expressed more clearly.

In any event, we do not at present validate the Rancher data either, which is a mistake IMHO. As such I'd say we proceed with this to help meet the tight timeline for mvp. However, #20 should remain open and there should be a follow up PR that addresses #20 by including data that can be validated.

Reply:
I saw the comment but was lost until I dig into what regionsrv-client is doing. The comment assumes that anyone picking up the issue should know those details and where to find them.

Reply:
The requirement for signature/checksum is now explicit in the issue. As I'm not familiar with regionsrv-client I don't know if the data can be amended or if the adapter has to dump exactly what it gets from metadata. This can be discussed in the issue.